index.html

```diff
@@ -33,7 +33,7 @@
 
     <section id="home">
         <div style="background-color: #f8f6e7;">
```

Comment:
Personally, I prefer if the styles stay in the stylesheet as opposed to being inline. If all styles are in a stylesheet, it's easy to tell where they are and prevents repetition, and it also makes it easier for elements that have lots of styles.

Reply:
I agree